src/status_im/contexts/chat/group/details/view.cljs

```diff
@@ -56,7 +70,9 @@
         chat-id                    (quo.context/use-screen-params)
         {:keys [admins] :as group} (rf/sub [:chats/chat-by-id chat-id])
         current-pk                 (rf/sub [:multiaccount/public-key])
-        admin?                     (get admins current-pk)]
+        admin?                     (get admins current-pk)
+        selected-contacts-count    (rf/sub [:group-chat/manage-members-count chat-id])
+        [has-error? set-has-error] (rn/use-state false)]
```

Comment:
Hooks should be before anything, even before re-frame subs

Reply:
@ulisesmac could you elaborate why?

Reply:
@flexsurfer 
Just React Hooks rules:
https://react.dev/reference/rules/rules-of-hooks#only-call-hooks-at-the-top-level

as a general good practice, we should (whenever possible) define the hooks first, so it's easy to track them and we make sure they are used before any possible invalid statement.

:thinking: I always use this pattern, but now I'm wondering if it really makes sense for the rest of the team. cc: @ilmotta  wdyt?

Reply:
I had the same question as @flexsurfer. I never thought of ordering subscriptions and hooks in a particular way, except for ordering them as naturally as the binding names should feel reading top to bottom. If there's no practical issue with using subs after hooks I think we can ignore this as a kind of guideline @ulisesmac

Reply:
OK! Sorry for the confusion added @ajayesivan